hack/make/cross

```diff
@@ -28,6 +29,12 @@ for platform in $DOCKER_CROSSPLATFORMS; do
 			export LDFLAGS_STATIC_DOCKER="" # we just need a simple client for these platforms
 			export BUILDFLAGS=( "${ORIG_BUILDFLAGS[@]/ daemon/}" ) # remove the "daemon" build tag from platforms that aren't supported
 		fi
+		# make windows cross compile static daemon work
+		if [ "${platform}" = "windows/amd64" ]; then
+			export CC=x86_64-w64-mingw32-gcc
+			export CGO_ENABLED=1
+			export LDFLAGS_STATIC_DOCKER="${LDFLAGS_STATIC_DOCKER/-linkmode external/} -extld=${CC}"
+		fi
```

Comment:
ok i added but let me know if you want me to move it

On Mon, Aug 10, 2015 at 4:13 PM, Tianon Gravi <notifications@github.com>
wrote:

> In hack/make/cross
> <https://github.com/docker/docker/pull/15459#discussion_r36695892>:
>
> > @@ -28,6 +29,12 @@ for platform in $DOCKER_CROSSPLATFORMS; do
> >  			export LDFLAGS_STATIC_DOCKER="" # we just need a simple client for these platforms
> >  			export BUILDFLAGS=( "${ORIG_BUILDFLAGS[@]/ daemon/}" ) # remove the "daemon" build tag from platforms that aren't supported
> >  		fi
> > +		# make windows cross compile static daemon work
> > +		if [ "${platform}" = "windows/amd64" ]; then
> > +			export CC=x86_64-w64-mingw32-gcc
> > +			export CGO_ENABLED=1
> > +			export LDFLAGS_STATIC_DOCKER="${LDFLAGS_STATIC_DOCKER/-linkmode external/} -extld=${CC}"
> > +		fi
>
> Hmm yeah, probably; just in case.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/15459/files#r36695892>.
>